src/app/open-source/page.tsx

```diff
@@ -19,23 +22,68 @@ export default function OpenSourcePage() {
+                                        <Duck colour="white" size={40} />
+                                        <Duck colour="white" size={40} />
+                                    </div>
+                                    <div className="relative flex flex-row">
+                                        <div className="w-12 bg-orange" />
+                                        <div className="relative flex flex-1 flex-col">
+                                            <span className="relative z-[2] bg-white py-4 pl-4 pr-16 text-2xl font-black text-black lg:pr-36 lg:text-3xl">
+                                                Who is the
+                                            </span>
+                                            <span className="relative z-[2] whitespace-nowrap border-2 border-white py-6 pl-4 pr-16 text-2xl font-bold underline lg:pr-36 lg:text-3xl">
+                                                Open Source Team?
+                                            </span>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div className="rounded-xl border-2 border-white px-4 py-6 md:w-[600px] md:px-6 md:py-6 lg:w-[500px]">
+                                    <p className="text-lg md:text-xl">
+                                        The{' '}
+                                        <span className="font-bold text-yellow">
+                                            CS Club Open Source Team
+                                        </span>{' '}
+                                        is a place for creative and inventive students who want to
+                                        build innovative open-source software together. We are a new
+                                        team currently working on a few projects, but we have many
+                                        more exciting projects planned for the future!
+                                    </p>
+                                </div>
+                                <div className="flex flex-row items-center">
+                                    <Image
+                                        src="/images/yellow-triangle.svg"
+                                        alt="Yellow Triangle"
+                                        className="mr-4"
+                                        width={40}
+                                        height={40}
+                                    />
+                                    <Link
+                                        className="text-lg font-bold underline md:text-3xl"
+                                        href="https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform"
+                                    >
+                                        Join the Open Source Team!
+                                    </Link>
```

Comment:
Make the text a bit bigger for `sm`-`md` widths.

```suggestion
                                    <Link
                                        className="text-lg font-bold underline sm:text-2xl md:text-3xl"
                                        href="https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform"
                                    >
                                        Join the Open Source Team!
                                    </Link>
```